troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.jsx

```diff
@@ -403,13 +415,35 @@ export default React.createClass({
     },
 
     onProjectCreateConfirm: function(name, description) {
-        this.viewBasic();
+        let sshKey = stores.SSHKeyStore.getAll();
+        if (sshKey) {
+            if (this.view != "IMAGE_VIEW" && sshKey.length === 0) {
+                this.view = "SSH_VIEW";
+                this.viewSSH();
```

Comment:
Makes sense.  I wonder if some sort of `this.setView()` function might be cleaner - a function that just sets the view to whatever is in `this.view`?  Outside the scope of this PR maybe.